core/server/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java

```diff
@@ -59,7 +59,7 @@
   public static final String GET_VERSION = "version";
   public static final String GET_METRICS = "metrics";
 
-  private final AlluxioWorker mWorker = AlluxioWorker.get();
+  private final AlluxioWorker mWorker = DefaultAlluxioWorker.get();
```

Comment:
as for your reason @aaudiber, I would argue that the DefaultAlluxioWorker and the main class should be decoupled